client/deposits/utils/index.ts

```diff
@@ -5,6 +5,9 @@ import { __ } from '@wordpress/i18n';
 import { dateI18n } from '@wordpress/date';
 import moment from 'moment';
 
+// Internal dependencies
```

Comment:
Nit: use the `/**` comment syntax for consistency.
```suggestion
/**
 * Internal dependencies
 */
```